security/elastic.py

```diff
@@ -109,16 +100,17 @@ def commit(self, region):
         if not region:
             raise StorageException("Missing region")
         try:
-            self.es.indices.create(index=self.index + region, body={
+            retval = self.es.indices.create(index=self.index + region, body={
                 "mappings": {
-                    "_default_": {
+                    "issue": {
                         "properties": {
-                            "region": {
-                                "type": "text",
-                            },
-                            "description": {
-                                "type": "text",
-                            },
+                            "id": {"type": "keyword"},
+                            "type": {"type": "keyword"},
+                            "description": {"type": "text"},
+                            "region": {"type": "keyword"},
+                            "tenant_id": {"type": "keyword"},
+                            "user_id": {"type": "keyword"},
+                            "data": {"type": "text"},
```

Comment:
not sure what you want to store there, but judging from name — would it be a good idea to store it as binary? https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html